components/sup/src/manager/mod.rs

```diff
@@ -164,7 +164,7 @@ impl Default for ManagerConfig {
             custom_state_path: None,
             eventsrv_group: None,
             update_url: "".to_string(),
-            update_channel: "".to_string(),
+            update_channel: ChannelIdent::default(),
```

Comment:
Somewhat tangentially, I think we can just derive `Default` for `ManagerConfig` now.

(Though perhaps getting a default update URL of something other than `""` should come before that 😬 )

Reply:
I'll leave that one to you as my list of follow on tasks is significant 😂